tests/dashbio_demos/app_circos.py

```diff
@@ -21,122 +20,885 @@
+                        ],
+                    },
+                },
+                {
+                    'type': 'SCATTER',
+                    'data': circos_graph_data['snp250'],
+                    'config': {
+                        'tooltipContent': {
+                            'source': 'block_id',
+                            'target': 'position',
+                            'targetEnd': 'value',
+                        },
+                        'innerRadius': 0.65,
+                        'outerRadius': 0.95,
+                        'strokeColor': 'grey',
+                        'strokeWidth': 1,
+                        'shape': 'circle',
+                        'size': 14,
+                        'min': 0,
+                        'max': 0.013,
+                        'axes': [
+                            {
+                                'spacing': 0.001,
+                                'start': 0.006,
+                                'thickness': 1,
+                                'color': '#4caf50',
+                                'opacity': 0.3,
+                            },
+                            {
+                                'spacing': 0.002,
+                                'start': 0.006,
+                                'thickness': 1,
+                                'color': '#4caf50',
+                                'opacity': 0.5,
+                            },
+                            {
+                                'spacing': 0.002,
+                                'start': 0.002,
+                                'end': 0.006,
+                                'thickness': 1,
+                                'color': '#666',
+                                'opacity': 0.5,
+                            },
+                            {
+                                'spacing': 0.001,
+                                'end': '0.002',
+                                'thickness': 1,
+                                'color': '#f44336',
+                                'opacity': 0.5,
+                            },
+                        ],
+                        'backgrounds': [
+                            {'start': 0.006, 'color': '#4caf50', 'opacity': 0.1},
+                            {
+                                'start': 0.002,
+                                'end': 0.006,
+                                'color': '#d3d3d3',
+                                'opacity': 0.1,
+                            },
+                            {'end': 0.002, 'color': '#f44336', 'opacity': 0.1},
+                        ],
+                    },
+                },
+            ],
+            size=700,
+        ),
+
+        'select-dataset-stack': dash_bio.Circos(
+            id='main-circos',
+            selectEvent={'0': 'hover'},
+            layout=[
+                {
+                    'id': 'chr9',
+                    'len': 8000000,
```

Comment:
Since this is displayed to the end user, isn't it clearer to write 'length'?

Reply:
It would be, but this is the way that the prop is written in React. We'd have to change it in the Circos component itself, so I think it's outside the scope of this PR. Correct me if I'm wrong!

Reply:
Ok, I see. Since I'm not familiar with the original components, I tend to underestimate all the legacy / inheritance we find ourselves with!